docs/src/main/sphinx/admin/properties-general.md

```diff
@@ -33,22 +33,6 @@ across nodes in the cluster. It can be disabled, when it is known that the
 output data set is not skewed, in order to avoid the overhead of hashing and
 redistributing all the data across the network.
 
-## `protocol.v1.alternate-header-name`
```

Comment:
The config is still there. Do we want to remove it @dain?

Reply:
Sorry... this is the result of a discussion with @electrum - I think we want to remove it in the docs for sure.. and potentially even in the code .. its flagged as deprecated there..